### PR TITLE
Slack notifications

### PR DESCRIPTION
**Pull request by sebastiancaraballo**
## What does this PR do?
Captura de Pantalla 2021-01-21 a la(s) 14 16 11
- This will send a notification to #open-source channel every Friday to the Engineering Metrics Open Source page.

Resolves [#EM-212](https://rootstrap.atlassian.net/browse/EM-212?atlOrigin=eyJpIjoiMzQ4NTI5ZjA2ODhjNDk1ZmI3MDRhZDlhZGYwYTVlN2UiLCJwIjoiaiJ9)

### CODE REVIEW

**Review by SandroDamilano — APPROVED** [review]
This is awesome!! 👏

File: app/services/slack_service.rb [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+  OPEN_SOURCE = '#open-source'.freeze
+  ENGINEERING_METRICS_URL = ENV['ENGINEERING_METRICS_URL']
+  EMOJI = ':wave:'.freeze
+
+  def open_source_reminder
+    message = I18n.t('services.slack.open_source_reminder.message',
+                     url: "#{ENGINEERING_METRICS_URL}/open_source")
+    ping(OPEN_SOURCE, message)
+  rescue Slack::Notifier::APIError => exception
+    ExceptionHunter.track(exception)
```
Comment (santib) [inline app/services/slack_service.rb:13]: doesn't ExceptionHunter track things automatically? cc @brunvez

File: spec/jobs/open_source_slack_job_spec.rb [hunk 2]
```diff
@@ -0,0 +1,13 @@
+require 'rails_helper'
+
+describe OpenSourceSlackJob do
+  describe '#perform' do
+    it 'calls SlackService' do
+      expect(SlackService)
+        .to receive(:open_source_reminder)
+        .once
```
Comment (santib) [inline spec/jobs/open_source_slack_job_spec.rb:8]: would it be better to test that there was a request call made to slack (using WebMock)?

File: spec/services/slack_service_spec.rb [hunk 3]
```diff
@@ -0,0 +1,41 @@
...
+    end
+
+    context 'when sent to valid channel' do
+      before { stub_notification_webhook }
+
+      it 'calls Slack Notifier' do
+        expect_any_instance_of(Slack::Notifier)
+          .to receive(:post)
+          .with(text: message, icon_emoji: SlackService::EMOJI)
+        subject
```
Comment (santib) [inline spec/services/slack_service_spec.rb:18]: same

**Review by santib — APPROVED** [review]
Good job!

File: app/services/slack_service.rb (hunk 1 above)
Comment (brunvez) [reply app/services/slack_service.rb:13]: yess, it should track them on jobs automatically, giving more context than doing it manually

**Pull request merged**